life_hc/MySQL/Vehicles/fn_chopShopSell.sqf

```diff
@@ -2,41 +2,32 @@
 /*
     File: fn_chopShopSell.sqf
     Author: Bryan "Tonic" Boardwine
+    Modified: Casperento
 
     This file is for Nanou's HeadlessClient.
 
     Description:
     Checks whether or not the vehicle is persistent or temp and sells it.
 */
-private ["_unit","_vehicle","_price","_cash","_ownerUnit"];
-_unit = [_this,0,objNull,[objNull]] call BIS_fnc_param;
-_vehicle = [_this,1,objNull,[objNull]] call BIS_fnc_param;
-_price = [_this,2,500,[0]] call BIS_fnc_param;
-_cash = [_this,3,0,[0]] call BIS_fnc_param;
-
-_ownerUnit = _unit getVariable "life_clientID";
+params [
+    ["_unit",objNull,[objNull]],
+    ["_vehicle",objNull,[objNull]],
+    ["_price",500,[0]]
+];
 
 //Error checks
 if (isNull _vehicle || isNull _unit) exitWith  {
-    life_action_inUse = false;
-    _ownerUnit publicVariableClient "life_action_inUse";
+    [] remoteExecCall ["life_fnc_chopShopSold", remoteExecutedOwner];
```

Comment:
Does this definitely work correctly on the HC? When these new RE* commands were new, there was issues on the HC because of them

Reply:
Yes, it works